src/logger/index.ts

```diff
@@ -227,6 +227,7 @@ export class Logger {
 
   error(error: RainbowError, metadata: Metadata = {}) {
     if (error instanceof RainbowError) {
+      if (error.name === 'AbortError') return;
```

Comment:
this will prevent all logs for abort calls to be sent to our sentry. Pretty necessary imo. Are there any other instances we want to avoid logging errors for? (timeouts, etc.??)